src/main/webapp/components/search-form-select/search-form-select.tsx

```diff
@@ -62,51 +93,70 @@ const Section = (props: SectionProps) => {
     </div>
   )
 }
-type SearchFormListProps = {
-  searchForms?: QueryType[]
-}
+
 const SearchFormList = (props: SearchFormListProps) => {
   const [filterText, setFilterText] = useState('')
-  const {
-    searchForms = [
-      { title: 'Search Form 1' },
-      { title: 'Search Form 2' },
-      { title: 'Another Search Form' },
-      { title: 'Yet Another' },
-    ],
-  } = props
+  const { searchForms, onSelect, userEmail } = props
 
   const searchFormsFilteredByText = searchForms.filter(searchForm =>
     matchesFilter({ filterText, str: searchForm.title || '' })
   )
-
   return (
-    <div style={{ padding: 10 }}>
+    <React.Fragment>
       <TextField
         value={filterText}
         onChange={e => setFilterText(e.target.value)}
         variant="outlined"
+        fullWidth
         placeholder="Type to filter"
       />
       <Section
-        searchForms={searchFormsFilteredByText}
+        onSelect={onSelect}
+        searchForms={searchFormsFilteredByText.filter(
+          form => form.metacard_owner === userEmail
+        )}
         startOpen
         title={'My Search Forms'}
       />
       <Section
-        searchForms={searchFormsFilteredByText}
+        onSelect={onSelect}
+        searchForms={searchFormsFilteredByText.filter(
+          form =>
+            form.metacard_owner !== userEmail &&
+            form.metacard_owner !== 'system'
+        )}
         title={'Shared Search Forms'}
       />
       <Section
-        searchForms={searchFormsFilteredByText}
+        onSelect={onSelect}
+        searchForms={searchFormsFilteredByText.filter(
+          form => form.metacard_owner === 'system'
+        )}
         title={'System Search Forms'}
       />
-    </div>
+    </React.Fragment>
   )
 }
 
-const SearchFormSelect = () => {
+export const SearchFormSelect = (props: SearchFormSelectProps) => {
   const [anchorEl, handleOpen, handleClose, isOpen] = useAnchorEl()
+  const { searchForms, loading, error, refetch, onSelect, userEmail } = props
```

Comment:
How about having the container handle the open state, as well as the loading, error, and refetch? Kind of like in query manager: https://github.com/connexta/revelio/blob/master/src/main/webapp/components/query-manager/query-manager.tsx#L56

It seems like this component shouldn't care about the loading/error state